source/stylesheets/base/_layout.scss

```diff
@@ -11,11 +11,41 @@ a {
   }
 }
 
-.layout {
+img {
+  max-width: 100%;
+}
+
+.split-layout {
   margin: $spacing-base;
   max-width: $page-max-width;
 
   @include breakpoint($breakpoint-medium) {
     margin: $spacing-large auto;
   }
 }
+
+.split-layout__section {
+  @include padding($spacing-base null);
+  display: flex;
+  flex-direction: column;
+
+  @include breakpoint($breakpoint-medium) {
+    @include padding($spacing-base);
+    flex-direction: row;
+  }
+}
+
+.split-layout__section-header {
+  flex: 1;
+  padding-right: $spacing-base;
+}
+
+.split-layout__section-content {
+  flex: 1;
+}
+
+.split-layout__divider {
```

Comment:
I'm wondering about this. 

You might be able to satisfy this solely with CSS, rather than need to add extra elements to your DOM. 

The best I can tell is when a `split-layout__section` directly follows another, we want to apply some spacing and a divider.